lib/index.js

```diff
@@ -1,6 +1,4 @@
 /*!
```

Comment:
confused by what's going on here with index.js and main.js, the pattern is the reverse of what is normally done.  By convention I believe the commonjs code is index.js and the browser/ES6 implementation is main.js.

Reply:
Updated - dad4dbf